dev/benchmarks/metrics_center/lib/google_benchmark.dart

```diff
@@ -0,0 +1,69 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:convert';
+import 'dart:io';
+
+import 'package:metrics_center/src/common.dart';
+
+const String _kTimeUnitKey = 'time_unit';
+
+const List<String> _kNonNumericalValueSubResults = <String>[
+  kNameKey,
+  _kTimeUnitKey,
```

Comment:
any reason why `_kTimeUnitKey` is not defined in the common?

Reply:
Because `_kTimeUnitKey` is specific to Google benchmark. If I find it to be used in other places, then I'll move it to common.